app/views/users/show.html.erb

```diff
@@ -1,20 +1,35 @@
 <!-- Placeholder for show -->
-<div>
-  <div>
-    <span><%= @user.name %></span>
-    <p>No of Posts: <%= @user.posts.count %></p>
+<div>      
+  <div class="user">
+    <div class= "user-photo">
+      <%= image_tag(@user.photo, alt: @user.name, class: "photo") %>
+      </div>
+      <div class="user-details">
+        <h1><%= @user.name %></h1>
+        <span>No of Posts: <%= @user.posts.count %></span>
+      </div>
+    </div>
+  <div class=bio>
+    <h2>Bio:</h2>
+    <h3><%= @user.bio %></h3>
+
   </div>
-  <hr />
-  <div>
-    <p>Bio: <%= @user.bio %></p>
-  </div>
-  <hr />
-  <div>
-    <% @user.posts.each do |post| %>
-      <p>Post #<%= post.id %>: <%= post.text %></p>
+  <div >
+    <% @user.posts.each do |post|%>
```

Comment:
- [ ] The posts that are rendered on the user show page should be the recent posts, not all of the user's posts. Kindly make use of the custom model method you created in a previous project.